libraries/botframework-connector/src/connectorApi/operations/conversations.ts

```diff
@@ -124,21 +124,21 @@ export class Conversations {
    * @param [options] The optional parameters
    * @returns Promise<Models.ConversationsSendToConversationResponse>
    */
-  sendToConversation(conversationId: string, activity: Models.Activity, options?: msRest.RequestOptionsBase): Promise<Models.ConversationsSendToConversationResponse>;
+  sendToConversation(conversationId: string, activity: Partial<Models.Activity>, options?: msRest.RequestOptionsBase): Promise<Models.ConversationsSendToConversationResponse>;
```

Comment:
This seems fine to me! But I want to bring up that the Activity contract should correctly define all the necessary information for a channel to properly process the Activity. If this holds, then changing to a `Partial` will open the door to customers sending malformed requests. Particularly customers using the `botframework-connector` library whom aren't using `botbuilder-core`.

Reply:
I agree with you in principle, though there are a few concerns I would describe as more pragmatic:

- The SDK currently makes no attempt to perform any validation short of `as Activity`
- Duplicating validation that already exists in the API layer couples our source code with the API which means updates have to happen in lock step

It's not an ideal change, but all the casting that exists currently is worse in my opinion.